packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
-        etiquetasH[u].innerHTML = esJseshtml.compile(etiquetasH[u].innerHTML);
-      }catch(error){
-        console.error(error);
-      }
-    }
-  }
-});
\ No newline at end of file
+import * as esJscore from 'https://esm.run/@es-js/core'
+import * as esJseshtml from 'https://esm.run/@es-js/eshtml'
+window.addEventListener('load', function () {
+  let etiquetas = document.getElementsByTagName('script')
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'etiquetas' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)